sw/tools/find_vpaths.py

```diff
@@ -3,55 +3,71 @@
 #It finds the correct VPATH to each source file from make, and then feeds all info to gcc -MM.
 import sys
 import os
-from os import path, getenv
+from os import path, getenv, environ
```

Comment:
how are these changes related to PR?

Reply:
In order to get a good qt project with a chibios based board, some changes were needed. Since they are in a clear seperate commit I did not bother creating a separate PR